airbyte-api/src/main/openapi/config.yaml

```diff
@@ -557,6 +557,44 @@ paths:
       responses:
         "204":
           description: Successful operation
+  /v1/connector_builder_projects/list:
+    post:
+      tags:
+        - connector_builder_project
+      summary: List connector builder projects for workspace
+      operationId: listProjects
```

Comment:
a naming nit: can this be listConnectorBuilderProjects to avoid confusion? Project sounds a pretty big word here and could cover many other things

Reply:
The create/delete/update operation ids are named the same way - I will adjust across the board.